lib/beaker/logger.rb

```diff
@@ -83,6 +83,25 @@ def initialize(*args)
 
       @destinations = []
 
+      @color_levels = options[:color_levels] || {
+        :error => RED,
+        :warn => BRIGHT_RED,
+        :success => MAGENTA,
+        :notify => BLUE,
+        :info => GREEN,
+        :debug => WHITE,
+        :trace => BRIGHT_YELLOW,
+        :perf => BRIGHT_MAGENTA,
+        :host => YELLOW
+      }
+
+      # Jenkins exposed variable - should be present on the slave directing
+      # the beaker run
+      ci_build = ENV['BUILD_NUMBER'] != nil
+
+      @color_levels[:notify] = NORMAL if ci_build
```

Comment:
I think this is probably much further than it's original intention, which is override colors for CI so they are easy to find when reviewing logs on Jenkins. Let's not scope creep this one much further. :+1: